### PR TITLE
 V-View Directive Added.

### PR DESCRIPTION
**Pull request by mylmz10**
There is one TODO. vnode.componentInstance is null on VueTest. It will be check again.

### CODE REVIEW

File: tests/unit/directives/ViewDirective.spec.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { StackLayout, Label, Nvw } from '../../../src/main';
+
+describe('Action Dialog Plugin Testing', () => {
+  const localVue = createLocalVue();
+  localVue.use(Nvw);
+
+  const ViewComponent = {
+    name: 'ViewComponent',
+    template: `
+          <div class="view-component-container">
```
Comment (yerol) [inline tests/unit/directives/ViewDirective.spec.js]: reformat please

File: tests/unit/directives/ViewDirective.spec.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      </div>
+    </ViewComponent>
+`,
+    components: {
+      ViewComponent,
+    },
+  };
+
+  //TODO vnode.componentInstance is null on VueTest. It will be check again
+  it(`The ActionDialog is shown on the document.`, () => {
```
Comment (yerol) [inline tests/unit/directives/ViewDirective.spec.js]: action dialog?

File: tests/unit/directives/ViewDirective.spec.js (hunk 1 above)
Comment (mylmz10) [reply tests/unit/directives/ViewDirective.spec.js]: Reformatted

File: tests/unit/directives/ViewDirective.spec.js (hunk 2 above)
Comment (mylmz10) [reply tests/unit/directives/ViewDirective.spec.js]: Copy paste :) Fixed

File: tests/unit/directives/ViewDirective.spec.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import { expect } from 'chai';
+import { mount, createLocalVue } from '@vue/test-utils';
+import { StackLayout, Label, Nvw } from '../../../src/main';
+
+describe('Action Dialog Plugin Testing', () => {
```
Comment (yerol) [inline tests/unit/directives/ViewDirective.spec.js]: Action Dialog?

File: tests/unit/directives/ViewDirective.spec.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  const localVue = createLocalVue();
+  localVue.use(Nvw);
+
+  const ViewComponent = {
+    name: 'ViewComponent',
+    template:
+      '<div class="view-component-container"><div ref="leftContent" class="left-content"></div>' +
+      '<div ref="rightContent" class="right-content"></div><slot /></div>',
```
Comment (yerol) [inline tests/unit/directives/ViewDirective.spec.js]: use template literal please

File: tests/unit/directives/ViewDirective.spec.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    template:
+      '<ViewComponent id="viewComponent"><div v-view:leftContent id="leftContentStack"><div>Left Content</div>' +
+      '</div><div v-view:rightContent><div>Right Content</div></div></ViewComponent>',
+    components: {
+      ViewComponent,
+    },
+  };
+
+  //TODO vnode.componentInstance is null on VueTest. It will be check again
+  it(`The ViewComponent is shown on the document.`, () => {
```
Comment (yerol) [inline tests/unit/directives/ViewDirective.spec.js]: you are not testing anything here.

File: tests/unit/directives/ViewDirective.spec.js (hunk 3 above)
Comment (mylmz10) [reply tests/unit/directives/ViewDirective.spec.js]: Changed on last commit.

**Pull request merged**